README.md

```diff
@@ -69,6 +69,30 @@ class PaymentsController < ApplicationController
 end
 ```
 
+## You can add more params:
+
+```ruby
+# app/controllers/payments_controller.rb
+class PaymentsController < ApplicationController
+  def create
+    payment = MonopayRuby::Invoices::AdvancedInvoice.new(
+      "https://example.com",                    # your redirect url
+      "https://example.com/payments/webhook"    # your webhook url
+    )
+
+    if payment.create(amount, destination: nil, reference: nil, other_merchant_paymant_info_params: {}, other_params: {})
```

Comment:
this creation move to variable, like `successful`

Reply:
and then check here, like `successful == true`

Reply:
I didn't catch you, could you, please, explain what you mean